### PR TITLE
Fallback to default icons if bad status is given

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: modules/mediawiki/files/favicon.php [hunk 1]
```diff
@@ -30,8 +30,17 @@ function streamFavicon() {
 
 	$status = $client->execute();
 	if ( !$status->isOK() ) {
-		header( 'HTTP/1.1 500 Internal Server Error' );
-		return;
+		$favicon = '/favicons/default.ico';
```
Comment (RhinosF1) [inline modules/mediawiki/files/favicon.php:33]: Any need to assign for a single use?

File: modules/mediawiki/files/favicon.php [hunk 2]
```diff
@@ -30,8 +30,17 @@ function streamFavicon() {
...
+		$url = wfExpandUrl( $favicon, PROTO_CANONICAL );
+		$client = MediaWikiServices::getInstance()
```
Comment (RhinosF1) [inline modules/mediawiki/files/favicon.php:35]: Could we use a more specific name here & for $status?

**Review by RhinosF1 — CHANGES_REQUESTED** [review]

File: modules/mediawiki/files/favicon.php (hunk 2 above)
Comment (Universal-Omega) [reply modules/mediawiki/files/favicon.php:35]: We don't need to. It's already used elsewhere, and this allows the rest to be executed. Just changing here won't work, and I want to maintain consistency.

File: modules/mediawiki/files/favicon.php (hunk 1 above)
Comment (Universal-Omega) [reply modules/mediawiki/files/favicon.php:33]: Code format. Looks neater to me.

**Pull request merged**